server/src/encrypt.py

```diff
@@ -0,0 +1,27 @@
+import jwt
+from cryptography.fernet import Fernet
+
+from src.settings import SECRET
+
+
+FERNET = Fernet(SECRET)
+
+
+def encrypt_password(msg: str) -> str:
+    return FERNET.encrypt(msg.encode()).decode('utf-8')
+
+
+def decrypt_password(msg: str) -> str:
+    return FERNET.decrypt(msg.encode('utf-8')).decode()
+
+
+def encrypt_jwt(**kwargs) -> str:
+    return jwt.encode(kwargs, SECRET, algorithm='HS256')
+
+
+def decode_jwt(jwt_token: str) -> dict:
+    return jwt.decode(jwt_token, SECRET, algorithms='HS256')
+
+
+def is_same_hash_password(hash_password: str, password: str) -> bool:
```

Comment:
just is_same_hash, it not attached to the password, and can be used in different ways.
Function will receive hashed message and message to compare, and you must hash message and compare is with hashed message.

If think in that way, encrypt_password -> encrypt, decrypt_password -> decrypt

What you think about that?

Reply:
we have a password and jwt for encryption and decryption, so I think the second word is to specify what exactly we want to encrypt or decrypt or add an argument to the jwt function that will be the value name False and then it will be possible to combine encrypt_password + encrypt_jwt -> encrypt(msg: str, **kwargs, jwt=False) and similarly with decrypt.

How about this?